src/App.css

```diff
@@ -1,42 +1,57 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+*, *::before, *::after {
+    box-sizing: border-box;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+body {
+    margin: 0;
+    background: linear-gradient(to right, #00AAFF, #00FF6C);
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+.calculator-grid {
+    display: grid;
+    margin-top: 2rem;
+    justify-content: center;
+    grid-template-columns: repeat(4,6rem);
+    grid-template-rows: minmax(7rem, auto) repeat(5, 6rem);
+}
+
+.calculator-grid > button {
```

Comment:
🟢 Example green comment: nothing too serious, optional code improvement, does not block a PR's merge